options/options.go

```diff
@@ -389,7 +406,7 @@ func (interp *interpreter) interpretFieldPseudoOptions(fqn string, fld *descript
 	}
 
 	// and process default pseudo-option
-	if index, err := interp.processDefaultOption(scope, fqn, fld, uo); err != nil && !interp.lenient {
+	if index, err := interp.processDefaultOption(scope, fqn, fld, uo); err != nil {
```

Comment:
We no longer check `interp.lenient` here (or above) because when lenience is enabled, the returned err will now always be nil.